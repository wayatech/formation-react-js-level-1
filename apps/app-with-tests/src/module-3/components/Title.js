import React from "react";
import PropTypes from "prop-types";

const Title = ({ title }) => <p style={{ fontWeight: "bold" }}>{title}</p>;

Title.propTypes = {
  title: PropTypes.string.isRequired,
};

export default Title;
