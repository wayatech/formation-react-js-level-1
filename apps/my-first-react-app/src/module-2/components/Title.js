import React from "react";

const Title = (props) => <p style={{ fontWeight: "bold" }}>{props.title}</p>;

export default Title;
