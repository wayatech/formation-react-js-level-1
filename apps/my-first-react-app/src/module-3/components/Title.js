import React from "react";

const Title = ({ title }) => <p style={{ fontWeight: "bold" }}>{title}</p>;

export default Title;
