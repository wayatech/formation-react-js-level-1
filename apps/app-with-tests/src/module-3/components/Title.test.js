import React from "react";
import renderer from "react-test-renderer";

import Title from "./Title";

test("Title display is consistent", () => {
  const component = renderer.create(<Title title="This is my title" />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
