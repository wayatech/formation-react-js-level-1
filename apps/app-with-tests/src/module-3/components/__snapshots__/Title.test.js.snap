// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title display is consistent 1`] = `
<p
  style={
    Object {
      "fontWeight": "bold",
    }
  }
>
  This is my title
</p>
`;
